### PR TITLE
Stippling

### PR DESCRIPTION
**Pull request by Preetesh21**
This PR is made for issue #1 
I have added the stipple.py file which takes an input image address and saves a stippled version of the same in the assets. I have also updated the Readme accordingly.

### CODE REVIEW

File: stipple.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Import the dependencies
+import argparse
+import PIL
+import random
+from PIL import Image
+from PIL import ImageColor
```
Comment (smaranjitghose) [inline stipple.py]: Both the imports can be in a single line

File: stipple.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import math
+import cv2
```
Comment (smaranjitghose) [inline stipple.py]: Why is this required?

File: stipple.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+args = vars(ap.parse_args())
+
+
+img = Image.open(args['image'])
+img = img.convert('LA')
+
+width, height = img.size
+imgNew = Image.new('LA', (width, height))
+
+print('generating...')
```
Comment (smaranjitghose) [inline stipple.py]: We do not need this. We are focussed with the terminal only for debugging or seeing progress bars (using tqdm)

File: stipple.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import cv2
+
+# construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+
+img = Image.open(args['image'])
+img = img.convert('LA')
```
Comment (smaranjitghose) [inline stipple.py]: Can you explain this

File: stipple.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+width, height = img.size
+imgNew = Image.new('LA', (width, height))
+
+print('generating...')
+# here performing the operation where we are randomly selecting the pixel and adding the effect.
+for x in range(width):
+    for y in range(height):
+        point = img.getpixel( (x,y) )[0]
+        randNum = random.randint(0, 260)
+        if randNum >= point:
```
Comment (smaranjitghose) [inline stipple.py]: Put this inside functions. Have a look at the other scripts, please. We want it to be as much as modularize as possible

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by Preetesh21** [comment]
Sir, I have implemented the changes you had asked and sorry for the mistakes. Since there is no ready to use functions in OpenCV for stippling so I have done my work in a user-defined function and called it in the code later on.

File: stipple.py (hunk 2 above)
Comment (Preetesh21) [reply stipple.py]: It is not, I was looking for some direct functionality so I had imported it but forget to remove.But now its not there

File: stipple.py (hunk 3 above)
Comment (Preetesh21) [reply stipple.py]: Removed.

File: stipple.py (hunk 4 above)
Comment (Preetesh21) [reply stipple.py]: It is converting the image into Greyscale('L') with alpha transparency('A') .Added to determine how a pixel is rendered when blended with another.

File: stipple.py (hunk 5 above)
Comment (Preetesh21) [reply stipple.py]: Done sir.
